📖  Add documentation files for 1.0 Bento components (#31907)
ampproject/amphtml: extensions/amp-youtube/amp-youtube.md

</amp-youtube>
```

### Migrating from 0.1

> Unlike `0.1`, the experimental `1.0` version does not automatically convert `data-param-controls` to `controls`, and takes `controls` directly as an attribute.

Reviewer: ```suggestion
The experimental `1.0` version of `amp-youtube` does not automatically convert `data-param-controls` to `controls`. Instead it takes `controls` directly as an attribute.
```

Author: Updated, thanks!
